koku/api/report/all/openshift/provider_map.py

```diff
@@ -211,11 +211,19 @@ def __init__(self, provider, report_type):
                         },
                         'count': None,
                         'delta_key': {'usage': Sum('usage_amount')},
-                        'filter': [{
-                            'field': 'product_family',
-                            'operation': 'contains',
-                            'parameter': 'Storage'
-                        }, ],
+                        'filter': [{}],
+                        'or_filter': [
```

Comment:
I look at this as a sign that maybe we need to re-think this part of the map structure.

It's fine for this PR, but can you also create a new issue, please? I think we should take another look at the filter portion of this mapping to see if there's a bettter way to organize this.